pharmd-app/src/components/Basic/GridCard.js

```diff
@@ -6,21 +6,12 @@
  * Date: 04-23-2020
```

Comment:
Do we still need this?

Reply:
What is the issue that you are referring to?

Reply:
I don't think GridCard's adjust to the height. The issue still exist :/  
Since we currently don't need height adjustments: this issue is not important. 
Lol let's fix the TODO spelling and leave it for future reference.

Reply:
with all these TODOs, would it be better to convert them into issues? i think having them as issues would be easier for visibility vs having them scattered throughout the code